content/en/blog/_posts/steering-committee-results-2021

```diff
@@ -0,0 +1,51 @@
+---
+layout: blog
+title: "Announcing the 2021 Steering Committee Election Results"
+date: 2020-11-08
```

Comment:
@kaslin @alisondy @onlydole @chris-short 
I believe the date is incorrect, it may not  publish correctly
```
date: 2020-11-08
```

Reply:
Oh! Great call, @reylejano! @kaslin if you submit a PR, I can approve those changes too!

Reply:
@onlydole  Follow-up PR https://github.com/kubernetes/website/pull/30394